Jasmine/Levels/ViewControllers/LevelImportViewController.swift

```diff
@@ -0,0 +1,114 @@
+import UIKit
+
+class LevelImportViewController: UIViewController {
+
+    // MARK: Constants
+    private static let actionSheetPhrases = "View Phrases"
+
+    fileprivate static let segueToPhrasesExplorer = "SegueToPhrasesExplorer"
+
+    // MARK: Layouts
+    fileprivate var levelCollectionView: GameLevelListViewController!
+
+    // MARK: Properties
+    fileprivate var viewModel: LevelImportViewModelProtocol!
+
+    fileprivate var selectedLevel: GameInfo!
```

Comment:
can this remain with the new VM? or need to use an optional tuple `selectedLevel: (index: Int, isDefault: Bool)?` ?